src/components/SchemaFormItem.tsx

```diff
@@ -135,9 +142,20 @@ export default defineComponent({
                   v-model={formItemValue.value}
                   lang="sql"
                   disabled={isDisabled}
+                  {...customProps}
                 />
               </div>
             )
+          } else if (isTemplate) {
```

Comment:
Since the value is already returned in the above `if` statement, it is recommended to use `if` here instead of `else if`.